esmvalcore/cmor/_fixes/cmip5/gfdl_cm2p1.py

```diff
@@ -56,6 +56,32 @@ def fix_data(self, cube):
         return cube
 
 
+class Sit(Fix):
+    """Fixes for sit"""
+
+    def fix_metadata(self, cubes):
+        """
+        Fix metadata.
+
+        Fixes wrong units.
+
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+
+        Returns
+        -------
+        iris.cube.Cube
+
+        """
+        cube = self.get_cube_from_list(cubes)
+        time = cube.coord('time')
+        if time.bounds.max() > 1e8:
+            time.bounds = None
+            time.guess_bounds()
+        return cubes
```

Comment:
Did you check that this procedure leads to the same time bounds as in unaffected variables?

Reply:
No, I do not check. And you are right I should, time bounds are usually not as easy to guess as lat - lon ones in regular grids

Reply:
Not, and in fact they are wrong. Fixing them manually now and restricting the fix to monthly data

Reply:
Done!